bookdown/04-technical/04-technical.Rmd

```diff
@@ -2,7 +2,8 @@
 
 This chapter gives an overview about technical details of the `r mlr_pkg("mlr3")` framework.
 
-At first, some details about parallelization and the usage of the `r cran_pkg("future")` are given.
+At first, some details about [Parallelization](#parallelization) and the usage of the `r cran_pkg("future")` are given.
+Parallelization refers to running multiple tasks simultaneously and is used to minimize necessary computer power. 
```

Comment:
Even though "tasks" is not wring here, it might clash with the mlr meaning of "task". Maybe its better to use "jobs"?